core/types/celo_dynamic_fee_tx.go

```diff
@@ -10,19 +24,21 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-// CeloDynamicFeeTx represents a CIP-64 transaction.
-type CeloDynamicFeeTx struct {
-	ChainID    *big.Int
-	Nonce      uint64
-	GasTipCap  *big.Int
-	GasFeeCap  *big.Int
-	Gas        uint64
-	To         *common.Address `rlp:"nil"` // nil means contract creation
-	Value      *big.Int
-	Data       []byte
-	AccessList AccessList
+const CeloDynamicFeeTxType = 0x7c
```

Comment:
i'd move this back to the list on transaction.go; Since that's the list of all tx types, clearer to see it there than here

Reply:
op-geth have already started putting their types in separate files see `deposit_tx.go` so it's not the list of all the types. Do you still think we should move this back?

Reply:
ok, didn't know, then let's follow their lead